### DISCUSSION TITLE
¡ayuda nuevo en github!

### DISCUSSION THREAD

**Discussion by glucambio** [community/community#121281] [New to GitHub]
Hola amigos estoy recien empezando a usar github realmente estoy empezando en el mundo de la programacion y estoy configurando el github, pero cuando escribo este comando git config --list  me genera el siguiente error y no se como solucionarlo.
credential.helper=osxkeychain
user.name=glucambio
user.email=gcolinalucambio@gmail.com
credential.helper=
credential.helper=/usr/local/share/gcm-core/git-credential-manager
credential.https://dev.azure.com.usehttppath=true
(END)

**Comment by whay333** [comment]
no veo cual es el error @glucambio, si eso es lo que realmente debe salir cuando usas git config --list
si deseas configurar esos parametros los puede hacer así:
git config --global user.name "tu nombre"
git config --global user.email "tu correo"


existen otros comandos que puedes encontrar en:
https://git-scm.com/book/es/v2/Personalizaci%C3%B3n-de-Git-Configuraci%C3%B3n-de-Git
o
https://git-scm.com/book/es/v2/Inicio---Sobre-el-Control-de-Versiones-Configurando-Git-por-primera-vez

espero te sirva, saludos
